.github/workflows/changelog-v3.yml

```
@@ -1,25 +1,238 @@
+            // Read the fixed changelog content
+            const fs = require('fs');
+            let fixedContent = '';
+            try {
+              fixedContent = fs.readFileSync('docs/src/content/docs/changelog.mdx', 'utf8');
+            } catch (error) {
+              fixedContent = 'Error reading fixed changelog content';
+            }
+            
+            message = `## ⚠️ Changelog Validation Issue
+
+@${author} Your PR contains changelog entries that were added to already-released versions. These need to be moved to the \`[Unreleased]\` section.
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 203: found character that cannot start any token

(syntax-check)